upload-exclude.txt

```diff
@@ -1,5 +0,0 @@
-/angular-app/**/*.js
```

Comment:
The upload-exclude.txt and include.txt files were part of a previous generation of deployment scripts that used rsync for code deployment from TeamCity.  Safe to remove.